lib/factory-helper.rb

```diff
@@ -157,26 +157,26 @@ def rand_in_range(from, to)
   end
 end
 
-require 'faker/address'
-require 'faker/code'
-require 'faker/company'
-require 'faker/finance'
-require 'faker/internet'
-require 'faker/lorem'
-require 'faker/name'
-require 'faker/team'
-require 'faker/phone_number'
-require 'faker/business'
-require 'faker/commerce'
-require 'faker/version'
-require 'faker/number'
-require 'faker/bitcoin'
-require 'faker/avatar'
-require 'faker/date'
-require 'faker/time'
-require 'faker/number'
-require 'faker/hacker'
-require 'faker/app'
+require_relative 'factory-helper/address'
+require_relative 'factory-helper/code'
+require_relative 'factory-helper/company'
+require_relative 'factory-helper/finance'
+require_relative 'factory-helper/internet'
+require_relative 'factory-helper/lorem'
+require_relative 'factory-helper/name'
+require_relative 'factory-helper/team'
+require_relative 'factory-helper/phone_number'
+require_relative 'factory-helper/business'
+require_relative 'factory-helper/commerce'
+require_relative 'factory-helper/version'
+require_relative 'factory-helper/number'
+require_relative 'factory-helper/bitcoin'
+require_relative 'factory-helper/avatar'
+require_relative 'factory-helper/date'
+require_relative 'factory-helper/time'
+require_relative 'factory-helper/number'
+require_relative 'factory-helper/hacker'
+require_relative 'factory-helper/app'
 
 require 'extensions/array'
```

Comment:
I need to change this to `require_relative` as well to make the gem work locally. Ideas?